extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/user/UserService.java

```diff
@@ -86,14 +87,18 @@ private void putAllUsers(Map<String, User> users, LDAPConnection ldapConnection,
             String usernameAttribute) throws GuacamoleException {
 
         try {
+	    //change search limits
```

Comment:
For single line comments, we like to start with a capital letter, and leave a space after the slashes:

// This is a friendly comment